src/postgres/src/include/pg_yb_utils.h

```diff
@@ -906,11 +906,13 @@ OptSplit *YbGetSplitOptions(Relation rel);
 	} while (0)
 #endif
 
-void ProcSetNodeUUID(const char *);
+void ProcSetTopLevelNodeId(const uint64_t *);
 void ProcSetTopRequestId(const uint64_t *);
 
-void
-top_level_request_id_uint_to_char(char *top_level_request_id, const uint64_t top_level_request_id_uint[2]);
+void uint128_to_char(char *char_id, const uint64_t uint_id[2]);
```

Comment:
Fix order of input/output params. 

Input params should go before the output params in the order.

Reply:
Even simpler would be to just return the output instead of passing in a pointer.

Reply:
Returning the output would require to allocate memory using malloc/palloc and then deallocating later to avoid memory leak. Passing the pointer seems a simpler approach.